sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -872,6 +873,73 @@ public long count(CaseCriteria caseCriteria) {
 		return result;
 	}
 
+	public List<DashboardQuarantineDataDto> getCaseInQuarantineForDashBoard(
```

Comment:
Please rename to `getQuarantineDataForDashboard` or something similar as you don't return case

Reply:
changed